src/components/Network/Network.vue

```diff
@@ -104,32 +132,106 @@ export default {
       ],
       ingresses_headers: [
         {
-          text: 'Name',
+          text: 'Hosts',
+          value: 'hosts',
           algin: 'start',
-          sortable: false,
+          sortable: true
+        },
+        {
+          text: 'Status',
+          value: 'status'
+        },
+        {
+          text: 'Name',
           value: 'name'
         },
+        {
+          text: 'Annotations',
+          value: 'annotations'
+        },
         {
           text: 'TLS',
           value: 'tls'
         },
         {
-          text: 'Hosts',
-          value: 'hosts'
+          text: 'Paths',
+          value: 'path'
+        },
+        {
+          text: 'Service',
+          value: 'services'
+        },
+        {
+          text: 'Created at',
+          value: 'creation_timestamp'
         }
       ]
     }
   },
-  mounted () {
+  beforeMount () {
     if (this.services_data.length === 0) {
       this.$store.dispatch('services/request_services')
     }
     if (this.ingresses_data.length === 0) {
       this.$store.dispatch('ingresses/request_ingresses')
     }
+  },
+  mounted () {
+    this.check_available()
+  },
+  methods: {
+    check_availablity (data) {
```

Comment:
Couldn't this function be also placed in the ingresses store? So the functionality/ logic would be stored in one file.

Reply:
And if we were to do so, we wouldn't need this variable 'ingress' and we could just use ingresses_data and everything would be consistent

Reply:
inside component or refresh button. it is for now in ingress.js